cloud/deploys/bin/azure/lib.sh

```diff
@@ -71,8 +71,79 @@ function azure::add_generated_secrets() {
   for key in "$@";
   do
     echo "Generating secret: ${key}"
-    local SECRET_VALUE="$(head /dev/urandom | LC_CTYPE=C tr -dc "${CHARSET}" | cut -c -40)"
+    local SECRET_VALUE="$(head /dev/urandom | LC_CTYPE=C tr -dc "${CHARSET}" |LC_CTYPE=C cut -c -40)"
     echo "Setting secret: ${key}"
     azure::add_secret "${VAULT_NAME}" "${key}" "${SECRET_VALUE}"
   done
 }
+
+#######################################
+# Get a secret value from the key vault
+# Arguments:
+#   1: The name of the key vault
+#   2: The name of the secret (used to identify it e.g. "postgres-password")
```

Comment:
suggestion (blocking): please include info about what the output looks like in the doc comment